lte/gateway/python/magma/pipelined/app/gy.py

```diff
@@ -137,8 +137,9 @@ def _deactivate_flows_for_subscriber(self, imsi, ip_addr):
             self._datapath,
             imsi,
         )
-        self._qos_mgr.remove_subscriber_qos(imsi)
-        self._remove_he_flows(ip_addr, None)
+        if self._qos_mgr:
```

Comment:
@pshelar Should we add a warning log or something when we hit this case? Do we expect qos manager to always be not None?

Reply:
@koolzz any idea why _qos_mgr would be null here?
I see the check in rpc-servicer for enforcement readiness. not sure how _qos_mgr could still be None here.

Reply:
Yeah also not sure, we check if it was initialized before calling deactivate. Could the qos mgr init call fail for any reason?

Reply:
I would suggest to move the discussion about archtiecture and design to https://github.com/magma/magma/issues/10524, as it feels out of scope for this PR.
I am convinced that with the more general discussion on https://github.com/magma/magma/issues/10524, we can merge this PR to move forward with the Sentry topic.